evm/utils/datatypes.py

```diff
@@ -55,6 +55,11 @@ def _get_top_level_keys(overrides: Dict[str, Any]) -> Iterator[str]:
     for prop in overrides:
         yield _extract_top_level_key(prop)
 
+# Dynamic subclassing is not supported by mypy
+# https://github.com/python/mypy/wiki/Unsupported-Python-Features
+# Most of the cases where we silence mypy boil down to cases where
+# dynamic subclasses where generated through this method
+
```

Comment:
I'm not sure I like the idea of having a blanket comment like this; I'd prefer if we had a separate, specific, comment for every place we need to silence mypy

Reply:
I've put a short comment on the places where we had to add a `ignore` because of this and then referenced to here to lookup this longer explanation. I thought that's a good trade off so that I don't have to put a lengthy explanation on each `ignore`